packages/server/lib/socket-base.ts

```diff
@@ -164,6 +163,7 @@ export class SocketBase {
     const cdpIo = this._cdpIo = this.createCDPIo(socketIoRoute)
 
     automation.use({
+      // @ts-ignore - this error is new, but not introduced in the most recent edit. TODO: fix
```

Comment:
I think this was causing issues with linting; VScode was showing an error here with the file open, but lint only started failing against it when I changed it. For ref,

`onPush` is defined as a `NullableMiddlewareHook = (() => void) | null`, which means either this function never receives the expected props, or `onPush` actually being `NullableMiddlewareHook` instead of something else is in question.

Reply:
This is all CDP/BiDi signature related so I would be surprised if the change was introduced here